advance-exercises/src/components/01-CompoundComponents.js

```diff
@@ -16,70 +9,125 @@
     - Arrow left, arrow up should select the previous option
 */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
 class RadioGroup extends React.Component {
   static propTypes = {
-    // defaultValue: PropTypes.string,                UN-COMMENT THIS LINE
-    children: PropTypes.shape().isRequired,
+    defaultValue: PropTypes.string,
```

Comment:
As the prop "defaultValue" is not "required" here, I think a default value (defaultProps) should be assigned to it.